app/routes/events/components/EventAdministrate/AdminRegisterForm.js

```diff
@@ -35,7 +35,9 @@ const AdminRegister = ({
           name="pool"
           component={SelectInput.Field}
           placeholder="Pool"
-          options={pools.map(pool => ({ value: pool.id, label: pool.name }))}
+          options={pools
+            .map(pool => ({ value: pool.id, label: pool.name }))
+            .concat([{ value: 0, label: 'Venteliste' }])}
```

Comment:
Maybe -1 is a more natural value than 0 when referring to a non existant pool? 0 is an indice in arrays etc – even though that have no effect here. I guess the option wont stick in the input if you use null or does not define it - right?